packages/authentication/types/allAuth.ts

```
@@ -0,0 +1,713 @@
+export interface GetPasswordResetResponse {
+  status: 200
+}
+
+export interface ResetPasswordRequest {
+  key: string
+  password: string
+}
+
+export interface ResetPasswordResponse {
+  // TODO
+}
```

Biome

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)